walk/config.go

```diff
@@ -82,6 +95,16 @@ func (cr *Configurer) Configure(c *config.Config, rel string, f *rule.File) {
 	if f != nil {
 		for _, d := range f.Directives {
 			switch d.Key {
+			case "generation_mode":
+				switch generationModeType(strings.TrimSpace(d.Value)) {
+				case generationModeUpdate:
+					wcCopy.updateOnly = true
+				case generationModeCreate:
+					wcCopy.updateOnly = false
+				default:
+					log.Printf("unknown generation_mode %q in //%s", d.Value, f.Pkg)
```

Comment:
@fmeum I added this print+continue since you last reviewed, so now it ignores invalid values...

Reply:
Can we fail here instead? It's often hard to spot these "warnings" in the output.

Reply:
Done 👍